### PR TITLE
Add new apps endpoint

### PR DESCRIPTION
**Pull request by seejohnrun**
Adds `find_installation_repositories_for_user`

Replacement for https://github.com/octokit/octokit.rb/pull/904

<strike>Will add VCR once we're settled.</strike>

### CODE REVIEW

File: spec/octokit/client/apps_spec.rb [hunk 1]
```diff
@@ -49,6 +49,14 @@
       end
     end # .installation
 
+    describe ".find_installation_repositories_for_user", :vcr do
```
Comment (kytrinyx) [inline spec/octokit/client/apps_spec.rb]: You shouldn't need to pass the `:vcr` option here.

**Comment by seejohnrun** [comment]
what do you think @kytrinyx ?

File: lib/octokit/client/apps.rb [hunk 2]
```diff
@@ -146,6 +146,19 @@ def remove_repository_from_integration_installation(installation, repo, options
         )
         remove_repository_from_app_installation(installation, repo, options)
       end
+
+      # List repositories accessible to the user for an installation
+      #
+      # @param installation [Integer] The id of a a GitHub App Installation
```
Comment (kytrinyx) [inline lib/octokit/client/apps.rb]: "a" is here twice.

**Comment by seejohnrun** [comment]
@kytrinyx Fixed that another another grammatical error that were present in multiple places in this file - thanks!

**Pull request merged**